recipe/meta.yaml

```diff
@@ -12,22 +12,19 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 3
+  number: 0
   skip: True  # [win]
 
 requirements:
   build:
-    - toolchain
     - pkg-config
-    - protobuf
-    - libprotobuf  # [linux]
+    - protobuf 3.4.*
     - ncurses 5.9*
     - zlib 1.2.*
```

Comment:
Sorry. Missed this. It should be [`zlib 1.2.11`]( https://github.com/conda-forge/conda-forge.github.io/blob/06e20200a735ca2a22848caa6f6ad73c0f8f7419/scripts/pin_the_slow_way.py#L96 ).